src/views/basics/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { FC } from "react";
+
+export const BasicsView: FC = ({}) => {
```

Comment:
Please delete this view and the basics page added above. We really do not need this, it was earlier part of the template